<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidgetPrintConfiguration">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Working hours</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTableView" name="tableViewWorkingHours"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Reminder Configuration</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="4" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item alignment="Qt::AlignTop">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Command</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Parameter</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QCheckBox" name="checkBoxUseDefaultBrowser">
               <property name="text">
                <string>Use default browser</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="spacing">
                <number>2</number>
               </property>
               <item alignment="Qt::AlignTop">
                <widget class="QLineEdit" name="lineEditCommand"/>
               </item>
               <item alignment="Qt::AlignTop">
                <widget class="QPushButton" name="pushButton">
                 <property name="text">
                  <string>Browse ...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditParameter"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPushButton" name="pushButtonDeleteAction">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Delete selected action</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item alignment="Qt::AlignBottom">
              <widget class="QPushButton" name="pushButtonAddAction">
               <property name="text">
                <string>Add new action</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QTableWidget" name="tableWidgetActions">
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>20</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>10</number>
           </attribute>
           <column>
            <property name="text">
             <string>Command</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Parameter</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="QTextEdit" name="textEdit">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Pause time! Stop working 5 to 10 minutes, go for a walk, relax your shoulders, your arms, yours hands and your fingers. Try to think about nice private things. Breath some fresh air. Come back with a fresh mind, and relaxed body. That way you will not damage your health and will improve your efficiency at work.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="labelRemindPauseTime">
             <property name="text">
              <string>Pause frequency</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QTimeEdit" name="timeEditRemindPauseTime">
             <property name="minimumSize">
              <size>
               <width>71</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>71</width>
               <height>22</height>
              </size>
             </property>
             <property name="time">
              <time>
               <hour>1</hour>
               <minute>0</minute>
               <second>0</second>
              </time>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="checkBoxRemindPauseTime">
           <property name="text">
            <string>Remind pause time</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" rowspan="2" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="labelDefaultAction">
             <property name="text">
              <string>Configured actions to be started every</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QTimeEdit" name="timeEditDefaultAction">
             <property name="minimumSize">
              <size>
               <width>71</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>71</width>
               <height>22</height>
              </size>
             </property>
             <property name="time">
              <time>
               <hour>3</hour>
               <minute>0</minute>
               <second>0</second>
              </time>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QCheckBox" name="checkBoxEnableDefaultActions">
             <property name="text">
              <string>Run periodically default actions</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignRight">
            <widget class="ClickableLabel" name="labelWhatIsThis">
             <property name="font">
              <font>
               <underline>true</underline>
              </font>
             </property>
             <property name="text">
              <string>What is this ? Please help !</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabPrintConfiguration">
        <attribute name="title">
         <string>Print Configuration</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>  Name configuration  </string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>User name: </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>Real name: </string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <widget class="QLineEdit" name="lineEditUserName">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEditRealName"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>  Image &amp;&amp; Footer  </string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QCheckBox" name="checkBoxDisplayImage">
                  <property name="text">
                   <string>Display image</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item alignment="Qt::AlignVCenter">
                   <widget class="QLabel" name="label_5">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>15</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Footer:</string>
                    </property>
                   </widget>
                  </item>
                  <item alignment="Qt::AlignTop">
                   <widget class="QTextEdit" name="textEditFooter">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>30</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="ClickableImage" name="OKButton">
         <property name="text">
          <string>OKButton</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableImage</class>
   <extends>QLabel</extends>
   <header location="global">Misc/clickableimage.h</header>
  </customwidget>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>Promoted/clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxUseDefaultBrowser</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEditCommand</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxEnableDefaultActions</sender>
   <signal>clicked(bool)</signal>
   <receiver>checkBoxUseDefaultBrowser</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxEnableDefaultActions</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEditParameter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxEnableDefaultActions</sender>
   <signal>clicked(bool)</signal>
   <receiver>pushButtonAddAction</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>726</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxRemindPauseTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>textEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>469</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxRemindPauseTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>timeEditRemindPauseTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>733</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxEnableDefaultActions</sender>
   <signal>clicked(bool)</signal>
   <receiver>timeEditDefaultAction</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>733</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
